### ISSUE TITLE
Can T2I-Adapter be used for SD-2.x？

### ISSUE THREAD

**Issue by rnwang04** [TencentARC/T2I-Adapter#19]
I wonder whether T2I-Adapter could be used for SD-2.x ?

**Comment by MC-E** [comment]
Because the number of channels is different between 1.x and 2.x, the adapter trained on 1.x can not be directly applied to 2.x. We are also training our adapters on the 2.x, and the result will be reported.

**Comment by thibaudart** [comment]
Have you some ETA for the 2.1?

**Comment by omaralvarez** [comment]
Yes, I am also interested on this. Would be awesome to know how is it coming along!

**Comment by AllenEdgarPoe** [comment]
Any updates?

**Comment by killah-t-cell** [comment]
Any updates?
